package.json

```diff
@@ -1,29 +1,32 @@
 {
-  "name": "randbot",
-  "version": "1.0.0",
-  "private": true,
-  "author": "hash-gaming",
-  "description": "Mostly productive but sometimes trolly",
-  "scripts": {
-    "linter": "eslint ."
-  },
-  "dependencies": {
-    "hubot": "2.19.0",
-    "hubot-diagnostics": "0.0.2",
-    "hubot-hello": "0.0.1",
-    "hubot-help": "0.2.2",
-    "hubot-pugme": "0.1.1",
-    "hubot-redis-brain": "0.0.4",
-    "hubot-rules": "0.1.2",
-    "hubot-scripts": "2.17.2",
-    "hubot-slack": "4.4.0"
-  },
-  "engines": {
-    "node": ">=8.0.0"
-  },
-  "devDependencies": {
-    "eslint": "4.8.0",
-    "eslint-config-airbnb-base": "12.0.2",
-    "eslint-plugin-import": "2.7.0"
-  }
+    "name": "randbot",
+    "version": "1.0.0",
+    "private": true,
+    "author": "hash-gaming",
+    "description": "Mostly productive but sometimes trolly",
+    "scripts": {
+        "linter": "eslint .",
+        "local-server-start": "docker-compose up --build --force-recreate",
+        "local-server-stop": "docker rm $(docker ps -a -q)",
```

Comment:
I'd rather this be `"local-server-stop": "docker-compose down"` because that won't remove all docker containers on your computer which is only needed as a cleanup step once you know that docker containers on your computer aren't needed anymore. `docker-compose down` just cleans up the resources that `docker-compose up` created.